developing-key4hep-software/SpackForDevelopers.md

```diff
@@ -5,6 +5,10 @@ However, it is not impossible and this is an area of spack that is currently und
 Unfortunately, this also means that the spack documentation might not be fully up-to-date on these topics.
 Hence, this page tries to collect some of the experiences the Key4hep developers have made.
 
+```{tip} 
+To obtain and setup `spack` take a look at {doc}`/spack-build-instructions-for-librarians/spack-setup`.
+```
```

Comment:
If it's between triple quotes it will appear as code but this is not code so those should be removed. Does that link work that way? Doesn't look like the usual markdown syntax

Reply:
This is what I found on how to do it with sphinx and when testing locally it rendered correctly.

Reply:
Ah right I missed the `{tip}` ha